storm-core/src/clj/backtype/storm/stats.clj

```diff
@@ -322,16 +307,16 @@
   [(.get_componentId global-stream-id) (.get_streamId global-stream-id)])
 
 (defmethod clojurify-specific-stats BoltStats [^BoltStats stats]
-  [(window-set-converter (.get_acked stats) from-global-stream-id symbol)
-   (window-set-converter (.get_failed stats) from-global-stream-id symbol)
-   (window-set-converter (.get_process_ms_avg stats) from-global-stream-id symbol)
-   (window-set-converter (.get_executed stats) from-global-stream-id symbol)
-   (window-set-converter (.get_execute_ms_avg stats) from-global-stream-id symbol)])
+  [(window-set-converter (.get_acked stats) from-global-stream-id identity)
```

Comment:
It is not necessary to convert the strings keys coming out from thrift deserialization, since we use them as strings anyway.

It is also confusing to debug, as the symbol `600` and the number `600` do not compare equally, yet they look exactly the same when printed out.

I changed this because in the course of writing the code, I printed the contents of a heartbeat and saw `600`, and when I tried to retrieve the value for the key `600` as a number, I got nothing back.